Competition2017/src/competition/subsystems/climbing/RopeAlignerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climbing;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class RopeAlignerSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(RopeAlignerSubsystem.class);
+    
+    public final XCANTalon intakeMotor;
```

Comment:
I don't see your point. We can either _hope_ it's safe or add the extra class to _make_ it safe; the latter has been done in other PRs and is working well with very little effort, so it doesn't make much sense to abandon that now.